### DISCUSSION TITLE
Infolist : relation between users

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#8821] [Help] [answered]
i have a relation between users and names , here it shows the id but i want to get the name , is it possible to do that in infolist filament v3
the provided code works with filament 2 without infolist feature 
```
Components\TextEntry::make('user_id')
        ->label(__('filament-notifications::translation.notifications.user'))
//      ->relationship('user', 'name')
        ->inlineLabel()
        ->hidden(fn (Notifications $record) => !($record->send_to == 'for_user')),
```

**Comment by leandrocfe** [comment]
`TextEntry::make('user.name')` ?

**Reply by Hegabovic** [reply]
i'm saving user_id but i need to display the name in the infolist

**Reply by leandrocfe** [reply]
ok. Do you have a `user` relationship on your Model?

**Comment by Hegabovic** [comment]
yup and it was working before using ifolist , but the method is not supported in ifolist , docs doesnt include relationship in infolist

**Reply by leandrocfe** [reply]
You can access the relationship using `TextEntry::make('user.name')`

**Reply by Hegabovic** [reply]
it is not working

**Reply by Hegabovic** [reply]
in edit form it is like that 
![Screenshot from 2023-10-02 14-13-05](https://github.com/filamentphp/filament/assets/56083879/407c5f07-b950-4653-a7e2-ac7f277bfd40)

**Reply by leandrocfe** [reply]
What isn't working? This was supposed to work. You have a `multiple` select, right? Shouldn't be `users` the relationship name?

**Reply by Hegabovic** [reply]
it is users.name , worked with users.name not user.name

**Reply by leandrocfe** [reply, marked as answer]
Sorry, I didn't understand your question..
